modules/openapi-generator/src/main/resources/python-blueplanet/app/{{packageName}}/util.mustache

```diff
@@ -91,24 +91,33 @@ def deserialize_datetime(string):
         return string
 
 
-def deserialize_model(data, klass):
+def deserialize_model(data, klass, attr_map=True):
     """Deserializes list or dict to model.
 
     :param data: dict, list.
     :type data: dict | list
     :param klass: class literal.
+    :param attr_map: defines if attribute_map is used in dict.
+    :type attr_map: bool
     :return: model object.
     """
     instance = klass()
 
+    # should be included in return types
     if not instance.swagger_types:
         return data
 
-    for attr, attr_type in six.iteritems(instance.swagger_types):
-        if data is not None \
-                and instance.attribute_map[attr] in data \
-                and isinstance(data, (list, dict)):
-            value = data[instance.attribute_map[attr]]
+    if data is None:
+        return instance
+
+    if not isinstance(data, (list, dict)):
+        return instance
```

Comment:
Why did you add this defensive protection about ingesting None and data not being list or dict?
Isn't it only called with list or dict inputs?
Should a ValueError be raised if invalid inputs are given rather than an empty instance of the model?

Reply:
@spacether Yes, i think that it's need to be clarified what is returned and when, ValueError could be raised but that would make it backwards incompatible. It's not up to me to decide but i can make the required changes.

Reply:
Good point ValueError could be added in a later PR when we include breaking changes.

Reply:
So for now keep  the old behaviour right?